files/fr/web/performance/lazy_loading/index.html

```diff
@@ -1,40 +1,40 @@
 ---
 title: Le chargement différé
 slug: Web/Performance/Lazy_loading
-translation_of: 'Web/Performance/Lazy_loading'
+translation_of: Web/Performance/Lazy_loading
 ---
-<p>Le <strong>chargement différé</strong> (en anglais <i lang="en">lazy loading</i>) est une stratégie d'identification des ressources non bloquantes (non critiques) afin de ne les charger qu'au moment où elles sont utiles. C'est une façon de raccourcir le <a href="/fr/docs/Web/Performance/Critical_rendering_path">chemin critique de rendu</a>, ce qui se traduit par une réduction du temps de chargement de la page.</p>
+<p>Le <strong>chargement différé</strong> (<em>lazy loading</em> en anglais) est une stratégie d'identification des ressources non bloquantes (non critiques) afin de ne les charger qu'au moment où elles sont utiles. C'est une façon de raccourcir le <a href="/fr/docs/Web/Performance/Critical_rendering_path">chemin critique de rendu</a>, ce qui se traduit par une réduction du temps de chargement de la page.</p>
```

Comment:
@tristantheb `lang` attributes are useful for screen reader users (allow to indicate the lang speech to use to vocalize the text). I'd not remove them, as it provides a better accessibility for these users.
Also, <em> is not the best choice here. We don't want to emphasize this text, we want to write it in italics to visually indicate that this text is in English.